frontend/src/components/filters/FilterInputNumber.js

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import {Input} from "antd";
+import FilterLabel from "./FilterLabel"
+import * as style from "./style"
+
+
+const FilterInput = ({
```

Comment:
Whether this is used or not, the name of a component should match the name of its file, especially if its a default export Component (meaning its the only component being exported in that file). Otherwise, this is possibly confusing, and in our case we have another component named 'Filterinput' too in FilterInput.js

Reply:
Corrected.